sklearn/impute.py

```diff
@@ -80,10 +95,10 @@ class SimpleImputer(BaseEstimator, TransformerMixin):
 
     Parameters
     ----------
-    missing_values : integer or "NaN", optional (default="NaN")
+    missing_values : real number, string, np.nan or None, \
+optional (default=np.nan)
```

Comment:
you can indent this for visual improvement.

Reply:
@jnothman this was on my request. If you indent, the spaces get included in the actual docstring. I know the above is ugly, but AFAIK the easiest way to have a too long type description with numpydoc

Reply:
indeed. I forget the discussion at https://github.com/numpy/numpydoc/issues/87, but I don't think this is mentioned there.

Reply:
I was also just looking back at that issue :-)

Reply:
Please replace these two lines with:
```
    missing_values : number, string, np.nan (default) or None
```
what we currently have here is unnecessarily verbose.